test/e2e/cloudstack_test.go

```diff
@@ -3923,7 +3691,7 @@ func TestCloudStackMulticlusterWorkloadClusterNewCredentialsSecretGitHubFluxAPI(
 		api.CloudStackToConfigFiller(
 			api.WithCloudStackCredentialsRef("test-creds"),
 		),
-		cloudstack.WithRedhat9Kubernetes126(),
+		cloudstack.WithRedhat9Kubernetes130(),
```

Comment:
does this need to be a higher k8s version? the original test uses 1.26 here rather than 1.25

Reply:
Yeah, I checked on this with Cavaughn. So the intention of these tests is to update the CS credential on workload cluster and the kubernetes version used doesn't really matter so wanted to just bump it to latest.